candies/cli/parsers/standard.py

```diff
@@ -103,37 +131,53 @@ def configured(new: Callable, command: Command, prefix: str = '.') \
         {'.command': 'tool', '..command': 'install', 'g': 'something'}.
     """
 
+    signature = inspect.signature(command.func)
     parser = new(command.name,
                  description=command.description,
-                 formatter_class=help(command))
+                 formatter_class=help(command),
+                 prefix_chars=prefixes(signature))
 
-    def short_of(x: Any) -> Optional[str]:
-        short = getattr(x, 'short', None)
+    def short_of(x: inspect.Parameter, kind: Any) -> Optional[str]:
+        short = getattr(kind, 'short', None)
 
         if short is None:
             # If `None`, then considered as omitted.
             return short
 
         if short == '':
-            raise ValueError(f"Flag '{parameter.name}' has invalid short name "
+            raise ValueError(f"Argument '{x.name}' has invalid short name "
```

Comment:
There is an article missing: "has _an_ invalid short name".

Reply:
Fixed.